Recipes/Recipes/View Controllers/RecipeDetailViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  RecipeDetailViewController.swift
+//  Recipes
+//
+//  Created by morse on 5/6/19.
+//  Copyright © 2019 Lambda Inc. All rights reserved.
+//
+
+import UIKit
+
+class RecipeDetailViewController: UIViewController {
+    @IBOutlet var textLabel: UILabel!
+    @IBOutlet var textView: UITextView!
+    
+    var recipe: Recipe? {
+        didSet {
+            updateViews()
+            print(recipe?.name)
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        updateViews()
+    }
+    
+    func updateViews() {
+        // Is this what was meant by "Check that the view is loaded when unwrapping the recipe object by using the view controller's isViewLoaded property."?
```

Comment:
It is! I would have written it 
```
if let recipe = recipe, isViewLoaded {
            textView.text = recipe.instructions
            textLabel.text = recipe.name
        }
```
which is functionally the same.
there are small differences between if-let and guard-let. For example, Guard requires an escape from current scope and creates a variable that is accessible outside the current scope, whereas if-let doesn't require a return and only creates an instance variable that is inaccessible outside the current scope. Many times it ends up being a matter of style, and since I don't need an early escape if the test fails, I choose the prettier, shorter solution...